packages/quill/src/core/subscriber.ts

```diff
@@ -20,25 +24,16 @@ interface Subscription {
 
 const subscribers = new WeakMap<object, Subscriber>();
 
-/**
- * Creates a Subscriber instance, and binds it to an object.
- */
-export function createSubscriber(object: Source): Subscriber {
-  const subscriber = new Subscriber();
-  subscribers.set(object, subscriber);
-  return subscriber;
-}
-
 /**
  * Gets the Subscriber instance bound to the given object.
- * Throws an error if the binding does not exist.
+ * Creates a new one if the binding does not exist yet.
  */
 export function getSubscriber(object: Source): Subscriber {
```

Comment:
Personally I'd be okay with this method returning undefined—the signature of _any_ getter from a map always has the potential to not find what you're looking for: 
```ts
interface WeakMap {
  get(key: object): undefined | any // value can be anything, including "not found"
}
```

At any rate, I'm okay with this direction too, but I think we should name the method slightly differently in this case to better reflect the fact that it's not _always_ just a getter. How about `findOrCreate()`?

Reply:
Renamed to `findOrCreateSubscriber()`